subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/JavaUtilLoggingSystemIntegrationTest.groovy

```diff
@@ -88,6 +88,33 @@ class JavaUtilLoggingSystemIntegrationTest extends AbstractIntegrationSpec {
     }
 
     @NotYetImplemented
```

Comment:
@w25r @bmuschko The implementation was causing issues during parallel execution. Most likely due to the static JUL log level setting. At that time, I did not have time to investigate further. This issue is still open since: https://github.com/gradle/gradle-private/issues/409